src/components/ProgressForm/ProgressForm.tsx

```diff
@@ -70,26 +68,29 @@ const ProgressForm = ({ questions }: formProps) => {
     };
 
     return (
-        <form className={styles.form}>
-            {questions.map((question, index) => (
-                <InputWithQuestions
-                    key={question.id}
-                    name={question.name}
-                    question={question.question}
-                    value={manager[index]}
-                    onChange={dispatch}
-                />
-            ))}
-            <button
-                className={styles.button}
-                onClick={(e) => handleSubmit(e)}
-                disabled={!isButtonEnabled}
-                type="submit"
-                data-testid="submit"
-            >
-                Submit
-            </button>
-        </form>
+        <>
+            <form className={styles.form}>
+                {questions.map((question, index) => (
+                    <InputWithQuestions
+                        key={question.id}
+                        name={question.name}
+                        question={question.question}
+                        value={manager[index]}
+                        onChange={dispatch}
+                    />
+                ))}
+                <button
+                    className={styles.button}
+                    onClick={(e) => handleSubmit(e)}
+                    disabled={!isButtonEnabled}
+                    type="submit"
+                    data-testid="submit"
+                >
+                    Submit
+                </button>
+            </form>
+            {isLoading && <Loader />}
```

Comment:
Why Loader is here?

Reply:
In previous implementation if we press submit the form contents would disappear in the loading state and re-appear when loading finished, which isn't a great UX (you can check the `before changes` video). Having the loader here will ensure the form stays when it's submitting progress.

Reply:
hmm getting you point, In my opinion the loader should be on button itself with being disabled state

Reply:
Sure! I'll do this implementation, thanks for the suggestion.